thoth/package_extract/image.py

```diff
@@ -464,6 +465,17 @@ def construct_rootfs(dir_path: str, rootfs_path: str) -> list:
     return layers
 
 
+def _get_absolute_link(root_path: str, path: str, iter: int) -> str:
+    """Find the absolute link of the given link."""
+    if iter > _MAX_SYMLINKS:
+        return None
+    elif os.path.islink(path):
+        next_path = os.path.normpath(os.path.join(root_path, os.readlink(path)))
+        return _get_absolute_link(root_path, next_path, iter + 1)
+    else:
+        return path
+
+
 def _get_python_interpreters(path: str) -> dict:
```

Comment:
The function returns a list, but the signature states it returns a dict. (We should start using mypy for type checking...).

Reply:
Thanks, changed.